components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/Connection.kt

```diff
@@ -61,6 +61,7 @@ internal interface Connection : Closeable {
 /**
  * A singleton implementation of the [Connection] interface backed by the Rust Places library.
  */
+@Suppress("TooManyFunctions")
```

Comment:
lol; i wonder if there is any value to these linters, we just keep suppressing them when they pop up.